api/src/main/resources/messages_fr.properties

```diff
@@ -71,3 +71,24 @@ ${project.parent.artifactId}.report.newEpisodesOfDiseases.females.label=F
 
 ${project.parent.artifactId}.report.lab.reportName=MSPP: Examens de laboratoire
 ${project.parent.artifactId}.report.lab.reportDescription=Rapport des Examens de laboratoire demandé par le MSPP à Haiti
+${project.parent.artifactId}.report.antenatal.reportName=Prise en charge de la mere
+${project.parent.artifactId}.report.antenatal.reportDescription=MSPP (Antenatal) report required by Haitian MoH to any clinic in Haiti
+${project.parent.artifactId}.report.antenatal.firstVisit.label = 1ière visite
+${project.parent.artifactId}.report.antenatal.secondVisit.label = 2ième visite
+${project.parent.artifactId}.report.antenatal.thirdVisit.label = 3ième visite
+${project.parent.artifactId}.report.antenatal.fourthVisit.label = 4ième visite
+${project.parent.artifactId}.report.antenatal.fifthVisit.label = 5ième visite et +
+${project.parent.artifactId}.report.antenatal.total.label = Total
+${project.parent.artifactId}.report.antenatal.gestationWeeks = semaines de grossesse
```

Comment:
Issues may arise coming from the space before the equal sign (` =`). Let's remove the spaces before `=`. Do you need the space before the property-value? If not, let's remove it as well. The same applies to the rest of the message properties.

Reply:
Done